src/Behat/Behat/Output/Node/Printer/Html/templates/footer.html

```diff
@@ -0,0 +1,186 @@
+</div><!-- /suite -->
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
+
+
+<script>
+
+    $(document).ready(function () {
+
+        /*
+
+         highlight v5
+
+         Highlights arbitrary terms.
+
+         <http://johannburkard.de/blog/programming/javascript/highlight-javascript-text-higlighting-jquery-plugin.html>
+
+         MIT license.
+
+         Johann Burkard
+         <http://johannburkard.de>
+         <mailto:jb@eaio.com>
+
+         */
+
+        jQuery.fn.highlight = function (pat) {
+            function innerHighlight(node, pat) {
```

Comment:
this function does not depend on the parent scope, so you don't need to redefine it for each highlighting call